app/src/androidTest/java/com/github/polypoly/app/commons/PolyPolyTest.kt

```diff
@@ -28,16 +35,22 @@ import kotlin.time.DurationUnit
 @RunWith(AndroidJUnit4::class)
 abstract class PolyPolyTest(
     private val clearRemoteStorage: Boolean, //> clear remote storage at the beginning of every test
-    private val fillWithFakeData: Boolean //> fill remote storage with fake data at the beginning of every test
+    private val fillWithFakeData: Boolean, //> fill remote storage with fake data at the beginning of every test
+    val signFakeUserIn: Boolean = false //> sign a fake user in at the beginning of every test
 ) {
     companion object {
         // Global tests constants
         const val TIMEOUT_DURATION = 5L
 
+        // Ensures only one global initialization even for multithreaded testing
+        var globalInitCompleted = false
+        var initLock = ReentrantLock(true)
```

Comment:
Where is this variable even used?

Reply:
This comes from the main while merging to resolve conflicts, it is not my code